src/services/lnd/index.ts

```diff
@@ -51,6 +53,101 @@ export const LndService = (): ILightningService | LightningServiceError => {
       .find((item) => item == pubkey)
   }
 
+  const invoiceProbeForRoute = async ({
+    decodedInvoice,
+    maxFee,
+  }: {
+    decodedInvoice: LnInvoice
+    maxFee: Satoshis
+  }): Promise<RawRoute | LightningServiceError> => {
+    if (!(decodedInvoice.amount && decodedInvoice.amount > 0))
+      return new LightningServiceError(
+        "No amount invoice passed to method. Expected a valid amount to be present.",
+      )
+    return _probeForRoute({
+      decodedInvoice,
+      maxFee,
+      amount: decodedInvoice.amount,
+    })
+  }
+
+  const noAmountInvoiceProbeForRoute = async ({
+    decodedInvoice,
+    maxFee,
+    amount,
+  }: {
+    decodedInvoice: LnInvoice
+    maxFee: Satoshis
+    amount: Satoshis
+  }): Promise<RawRoute | LightningServiceError> => {
+    if (!(amount && amount > 0))
+      return new LightningServiceError(
+        "Invalid amount passed to method for invoice with no amount. Expected a valid amount to be passed.",
+      )
+    return _probeForRoute({
+      decodedInvoice,
+      maxFee,
+      amount,
+    })
+  }
+
+  const _probeForRoute = async ({
```

Comment:
this is a private method but here is not necessary to `mark` with `_` the method because is not part of a calls and we are not exporting it. please just use `probeForRoute`